### ISSUE TITLE
Document supported docker run options

### ISSUE THREAD

**Issue by NathanUrwin** [lavie/runlike#52]
> Many docker run options aren't yet supported, but the most commonly used ones are.

Can you document what specific docker run options are supported and which ones aren't? Great project btw! 😄

**Comment by lavie** [comment]
Thanks, @NathanUrwin.
That's a good idea.

Until it's done, though, you can see the implemented options pretty clearly in this function: https://github.com/lavie/runlike/blob/0a5b8a274f5ff2dd611084acadff0c7883a92224/runlike/inspector.py#L155

**Comment by NathanUrwin** [comment]
Hey @lavie 👋 

Would that be all the `self.get_fact` calls?

```bash
$ grep 'self.get_fact' runlike/inspector.py
        values = self.get_fact(path)
        hostname = self.get_fact("Config.Hostname")
        user = self.get_fact("Config.User")
            mac_address = self.get_fact("Config.MacAddress") or {}
        ports = self.get_fact("NetworkSettings.Ports") or {}
        ports.update(self.get_fact("HostConfig.PortBindings") or {})
        links = self.get_fact("HostConfig.Links")
        restart = self.get_fact("HostConfig.RestartPolicy.Name")
            max_retries = self.get_fact(
        devices = self.get_fact("HostConfig.Devices")
        labels = self.get_fact("Config.Labels") or {}
        log_type = self.get_fact("HostConfig.LogConfig.Type")
        log_opts = self.get_fact("HostConfig.LogConfig.Config") or {}
        hosts = self.get_fact("HostConfig.ExtraHosts") or []
        image = self.get_fact("Config.Image")
        name = self.get_fact("Name").split("/")[1]
        network_mode = self.get_fact("HostConfig.NetworkMode")
        privileged = self.get_fact('HostConfig.Privileged')
        stdout_attached = self.get_fact("Config.AttachStdout")
        if self.get_fact("Config.Tty"):
        cmd_parts = self.get_fact("Config.Cmd")
```

Thanks for the quick reply!

**Comment by NathanUrwin** [comment]
I'm going to close this issue since you merged #53! Thanks for the good work! 😄
